src/olympia/amo/utils.py

```diff
@@ -477,32 +477,26 @@ def parse_html(tree):
     return serializer.render(stream)
 
 
-def resize_image(src, dst, size=None, remove_src=True, locally=False):
-    """Resizes and image from src, to dst. Returns width and height.
+def resize_image(src, dst, size=None):
```

Comment:
this is really just a nit… but I hate 3-char "words", can't we just name them `source` and `destination` (or `target` if it has to be shorter) maybe? Note that this might pull through the whole system so this is just me thinking loud without any need to change the current state :)